app/views/qsets/show.html.erb

```diff
@@ -1,22 +1,15 @@
 <div class="container-fluid" id="qset-show-container" data-qset-id="<%= @qset.id %>">
-  <%= render "qset_context" %>
-  <div id="generate-question-button-filter">
-    <%= render "generate_question" %>
+  <div class="show-large-screen">
+    <%= render "show_large_screen" %>
```

Comment:
the class in the outer div makes sense now, that's great. Why did the partial change to `show_large_screen` though? As a class, `.show-large-screen` clues the developer about the behavior of that element, but as a file/partial name `show_large_screen` isn't very helpful to indicate what's going to be rendered (or what's in the file). I'd switch the filenames back for greater readability.